src/rust/mqtt-publisher/src/main.rs

```diff
@@ -136,16 +137,26 @@ async fn eventpoll_loop(mut eventloop: EventLoop) -> Result<()> {
 struct PriceLevel {
     price: u128,
     size: u128,
+    txn_version: u128,
```

Comment:
- [ ] Update `mqtt` docs site page to reflect the new payload
- [ ] Also mention under the price levels entry on the docs site that you'll need to sett the `MQTT_PRICE_LEVELS` environment variable (it is already in the `example.env` file, of course, but who have set up deployments and then later wanted price levels didn't think to check): here you can link back to the DSS landing page and talk about when you copy the `exmaple.env` file for example